core/src/main/java/org/elasticsearch/common/Base64.java

```diff
@@ -161,6 +161,9 @@
  * @author rob@iharder.net
  * @version 2.3.7
  */
+
+// Given the class comment above ^^^^^ the license header on top of this file looks bogus
```

Comment:
should we have a followup issue for this? I don't think this is the only place we have this problem...